src/Table/index.jsx

```diff
@@ -20,10 +92,11 @@ class Table extends React.Component {
         <tr>
           {this.props.columns.map(col => (
             <th
+              className={this.props.tableSortable ? classNames({ sortable: col.columnSortable }) : ''}
```

Comment:
Flagging this line as a potential naive solution to [EDU-1535](https://openedx.atlassian.net/browse/EDUCATOR-1535) - if we're okay generalizing "all non-sortable column headings are hidden", we can change the else clause here to `'sr-only'`.

Is that a generalization we want to have? I'm worried we may eventually want to display a non-sortable heading.

Reply:
I think I could certainly see a case where someone would want to display a non-sortable heading, and I'd worry about inserting logic that benefits our page here at the expense of others interested in using the table, but I could be wrong.

Reply:
Cool, thanks for the confirmation. I was leaning that way as well, so I think I'll investigate getting a more specifically targeted approach working.